js/src/pages/price-benchmark/faq-link.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { ExternalLink } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { recordGlaEvent } from '~/utils/tracks';
+
+const URL = 'https://woocommerce.com/document/google-for-woocommerce/faq/';
+
```

Comment:
@ankitrox We are missing the documentation for @fires property for event tracking.